trade.lic

```diff
@@ -0,0 +1,262 @@
+=begin
+  Documentation: https://elanthipedia.play.net/Lich_script_repository#trade
+=end
+custom_require.call(%w(common common-items common-money common-travel drinfomon equipmanager))
+
+class Trade
+  include DRC
+  include DRCT
+  include DRCI
+  include DRCM
+
+  def initialize
+    @town_data = get_data('town')
+    @equipment_manager = EquipmentManager.new
+    settings = get_settings
+    @contract_container = settings.trade_contract_container
+    @hometown = settings.hometown
+    bput("open my #{@contract_container}", 'You open', 'That is already')
+    @equipment_manager.empty_hands
+    @caravan = OpenStruct.new
```

Comment:
If you're going to loop, this feel like something that should be in `run` instead of `initialize` (so it clears `@caravan` before each run)